src/components/common/FormInputError.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useMemo } from 'react'
+import styled from 'styled-components'
+
+import { FormMessage } from 'components/common/FormMessage'
+
+const Wrapper = styled.div<{ $margin?: string; $padding?: string }>`
+  margin: ${({ $margin = '0' }): string => $margin};
+  padding: ${({ $padding = '0' }): string => $padding};
+`
+
+export interface Props {
+  errorMessage?: string
+  schemaError?: boolean
+  visibilityHidden?: boolean
+  wrapperMargin?: string
+}
+
+export const FormInputError: React.FC<Props> = ({
+  errorMessage,
+  visibilityHidden = true,
+  wrapperMargin = '0.5rem 0',
+}) => {
+  const message = useMemo(() => {
+    if (errorMessage) {
+      return errorMessage
+    } else {
+      if (visibilityHidden) {
+        return 'Stop using developer tools! :D'
+      }
+
+      return null
+    }
+  }, [errorMessage, visibilityHidden])
```

Comment:
Don't use memo for a few if conditions

Reply:
I would swear I delete this memo, my memo is blurry right now about this

Reply:
Actually I think I also deleted some props we don't need, this might be in some local branch of mine...